src/pages/Admin/Details/ValidationSchema.ts

```diff
@@ -32,7 +32,8 @@ export interface RoomSchemaShape {
   title: string;
   venueName?: string;
   url?: string;
-  useUrl: boolean;
+  useUrl?: boolean;
+  image_url: string;
```

Comment:
Why not using `lowerCamelCase` for `image_url`?

Reply:
This is legacy code in the firebase, if I change it here. I'll have to change it in another 20-30 places. I don't think this should be done in this PR.

Reply:
Yup, agreed OOS for here. Just wanted to know the 'why'